tesseract_motion_planners/ompl/src/ompl_motion_planner.cpp

```diff
@@ -358,13 +358,7 @@ OMPLProblemConfig OMPLMotionPlanner::createSubProblem(const PlannerRequest& requ
   std::vector<std::string> active_link_names = manip->getActiveLinkNames();
 
   // Get Plan Profile
-  std::string profile = end_instruction.getProfile();
-  profile = getProfileString(name_, profile, request.plan_profile_remapping);
-  auto cur_plan_profile =
-      getProfile<OMPLPlanProfile>(name_, profile, *request.profiles, std::make_shared<OMPLDefaultPlanProfile>());
-  cur_plan_profile = applyProfileOverrides(name_, profile, cur_plan_profile, end_instruction.getProfileOverrides());
-  if (!cur_plan_profile)
-    throw std::runtime_error("OMPLMotionPlanner: Invalid profile");
+  auto cur_plan_profile = request.profiles->getProfile<OMPLPlanProfile>(name_, end_instruction.getProfile(name_));
```

Comment:
Do you have an explanation of how this works compared to the previous implementation?

Reply:
The same. `end_instruction.getProfile(name_)` attempts to get the profile name associated with the specific planner (i.e. `name_`); if that doesn't exist, the default profile name is returned. Then `getProfile` tries to find a profile of the specified type with the profile name returned. If it does not exist, it throws a `std::runtime_error`